docs/maintain-nominator.md

```diff
@@ -12,6 +12,36 @@ By doing so, you become a nominator for the validator of your choice. Pick your
 
 While your DOTs are staked by nominating a validator, they are 'locked' (bonded). You can receive new DOTs in your account but you cannot stake as validator or transfer DOTs away from your account. You can [un-nominate at any time](maintain-guides-how-to-unbond) to unlock your funds. Keep in mind that the un-nomination is effective in the next era.
 
+### What does active/inactive nomination mean?
```

Comment:
```suggestion
### Active vs. Inactive Nomination
```

Reply:
Headings should not be questions: https://writing.stackexchange.com/questions/14516/can-section-headings-in-a-paper-be-questions